src/nft-card-back.ts

```diff
@@ -189,7 +203,8 @@ export class NftCardBackTemplate extends LitElement {
         await this.requestUpdate()
       }
     })
-    // Console.log(this.shadowRoot.firstElementChild.offsetHeight)
+    this.cardHeight = this.shadowRoot.firstElementChild.offsetHeight
+    this.cardWidth = this.shadowRoot.firstElementChild.offsetWidth
```

Comment:
so if i set cardWidth from a parent component to e.g. 500px, what happens? it tries to do 500 and then changes it to whatever it ended up actually being?